src/components/flyout/drawer.js

```diff
@@ -30,7 +30,11 @@ class Drawer extends Component {
       <div className="drawer-wrapper">
         <div className="drawer-header" onClick={this.toggleHeader}>
           {this.props.title}
-          <img src={ChevronIcon} className="chevron-open" alt="ChevronIcon" />
+          {
+            this.state.showContent
+            ? <img src={ChevronIcon} className="chevron-close" alt="ChevronIcon" />

```

Comment:
"ChevronIcon" also isn't helpful. This string needs to be something that someone who is in some way impaired can make sense of from a functional point of view. Imagine that someone who is blind is navigating the site using a screen reader. When the reader reads "ChevronIcon", what does that tell them?